src/main/java/com/amazon/ion/impl/bin/WriteBuffer.java

```diff
@@ -1300,7 +1301,11 @@ public void writeVarUIntDirect2At(long position, long value)
         block.data[offset + 1] = (byte) ((value                            & VAR_INT_MASK) | VAR_INT_FINAL_OCTET_SIGNAL_MASK);
     }
 
-    public void writeUInt8At(final long position, final long value)
+    public void writeByteAt(final long position, final byte value) {
```

Comment:
🗺️ Convenience overload. If it's a perf problem we can remove it, but I bet the JIT is smart enough to optimize this out.